src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,112 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int INITIAL_CAPACITY = 10;
+    private static final double GROWTH_FACTOR = 1.5;
+    private Object[] array = new Object[INITIAL_CAPACITY];
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```

0 is default value for int